angularSTEP/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-toolbar color="primary" class="mat-elevation-z8">
```

Comment:
The indentation per the Google style guide we were given before (https://google.github.io/styleguide/htmlcssguide.html#General_Formatting_Rules) should be two spaces instead of four. I think this is the only html file in this PR that needs to be fixed.

Reply:
I see the paired SCSS file is also currently using 4-space indents, but IMO this is less of a problem if a given file is indented consistently